public/pathfindingAlgorithm.html

```diff
@@ -66,6 +66,7 @@
         <p class="clear-board-text" id="clear-board-text">Clear board</p>
         <p class="reset-board-text" id="reset-board-text">Reset board</p>
         <button class="btn btn-light" id="find-path-btn">Find Path!</button>
+        <p class="help-btn" id="help-btn"></p>
```

Comment:
why not use the button tag?

Reply:
Yes, I think using button tag make more sense.